pkg/openstack/instancesv2.go

```diff
@@ -79,6 +79,18 @@ func (os *OpenStack) instancesv2() (*InstancesV2, bool) {
 
 // InstanceExists indicates whether a given node exists according to the cloud provider
 func (i *InstancesV2) InstanceExists(ctx context.Context, node *v1.Node) (bool, error) {
+	if i.regionProviderID {
+		if node.Spec.ProviderID == "" {
+			klog.V(4).Infof("Instance %s should initialized first", node.Name)
```

Comment:
nit:
```suggestion
			klog.V(4).Infof("Node %s should be initialized first", node.Name)
```

Incidentally, it looks like you've started moving to [structured logging](https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1602-structured-logging/README.md) in at least the loadbalancer code.

This might be written as a structured log instead:
```golang
klog.V(4).InfoS("Node is not yet initialised", "node", klog.KObj(node))
```